ui/storybook/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "@perses-dev/storybook",
+  "version": "0.22.0",
+  "private": true,
+  "scripts": {
+    "storybook:start": "storybook dev --config-dir src/config --port 6006",
+    "storybook:clean": "rimraf storybook-static/",
+    "storybook:build": "npm run storybook:clean && storybook build --config-dir src/config",
+    "storybook:serve": "npx http-server ./storybook-static",
+    "lint": "eslint src --ext .ts,.tsx",
+    "lint:fix": "npm run lint -- --fix",
+    "type-check": "tsc --noEmit"
```

Comment:
Prefixing everything because the storybook is really different from our source code, and we don't want to run all of these steps in most cases where we're running things like "build" and it would be tedious to always remember to filter out the storybook workspace.